bin/build-galeforce.sh

```diff
@@ -14,13 +15,25 @@ function downloadBusybox() {
   fi
 }
 
+# go get github.com/ericchiang/pup
```

Comment:
I've put this in as an example. To work you'll have to install go and then go get pup, though its otherwise a really clean tool for command line html parsing.